.env

```diff
@@ -1 +1 @@
-DATABASE_URL=postgres://postgres:002128@localhost:5432/postgres
\ No newline at end of file
+DATABASE_URL=postgres://postgres:000000@localhost:5433/postgres
```

Comment:
Is this intended to be changed ?

Reply:
by right this file supposed to be on gitignore this is my own env file LOL I reinstalled postgres on my com the the password reset itself

Reply:
everyone's .env file supposed to be diff since its their own login details